renovate.json

```diff
@@ -2,20 +2,25 @@
   "extends": [
     "config:base"
   ],
-
   "pinVersions": false,
   "rebaseStalePrs": true,
+  "timezone": "America/Los_Angeles",
   "schedule": [
-    "after 9am and before 3pm"
+    "after 9am and before 3pm every tuesday"
   ],
+  "prConcurrentLimit": 5,
   "gitAuthor": null,
   "packageRules": [
     {
       "extends": "packages:linters",
       "groupName": "linters"
     }
   ],
-  "labels": ["dependencies"],
-  "prConcurrentLimit": 5,
-  "reviewersFromCodeOwners": true
+  "reviewersFromCodeOwners": true,
+  "labels": [
+    "dependencies",
+    "type: cleanup",
+    "priority: p1",
```

Comment:
Priority should be p2 - these upgrades are not as urgent when done periodically. We can bump it to p1 manually in cases when the upgrade fixes security vulnerability or similar issues.

Reply:
Updated!